manifests/server/puppetserver.pp

```diff
@@ -233,6 +234,13 @@
     }
   }
 
+  unless $::osfamily == 'Windows' {
```

Comment:
What's the reasoning to skip it on Windows? AFAIK we don't support the server on Windows anyway.

Reply:
I don't support it on Windows, either :)  I'll remove the 'unless'.